internal/pkg/cli/run_local.go

```diff
@@ -369,6 +369,14 @@ func (o *runLocalOpts) getTask(ctx context.Context) (orchestrator.Task, error) {
 		Containers: make(map[string]orchestrator.ContainerDefinition, len(td.ContainerDefinitions)),
 	}
 
+	if o.proxy {
```

Comment:
Why does pause need the default credentials only on `o.proxy` instead of always?

Reply:
we don't do anything with them in `pause` if we aren't creating proxy connections! we (still) always inject the creds into the workload containers.